.travis.yml

```diff
@@ -10,6 +10,10 @@ script:
   - cd tools/docker
   - sudo ./image-build.sh -b ubuntu:18.04
   - sudo ./build.sh map -f MSGLIB=zmq BUILD_TESTS=ON
+  # Start controller & agent node detached and check status
+  - sudo ./run.sh start-controller-agent -d -n controller
```

Comment:
Since it has controller and agent, it would be better to use `-n controller-agent`.